### DISCUSSION TITLE
Изображение (без названия файла)

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#232] [Как это сделать?] [answered]
1. Подскажите, как можно убрать подпись-название файла (в поле с типом файл image/*), чтобы осталось только само изображение в поле?

2. Почему файл иногда показывает две загрузки и ошибку, иногда нет, а иногда одну загрузку?

3. Можно ли обрезать изображения при загрузке  и уменьшать размер?

![Screenshot_2022-10-07-19-10-44-575_com android chrome](https://user-images.githubusercontent.com/113771444/194599547-250fc0cd-7516-4dd0-a830-5255d4bb10e4.jpg)
![Screenshot_2022-10-07-19-10-09-556_com android chrome](https://user-images.githubusercontent.com/113771444/194599566-0cf98ab4-f038-4aee-bc9a-ab1b76537077.jpg)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
1 и 3 не делали

2 двоение посмотрим. По умолчанию макс размер 10 мб.

**Reply by ttm-alexeypolunin** [reply]
Я думаю 3 будет платной API-функцией через наш сервер.

**Reply by mihail-cornev** [reply]
> 1 и 3 не делали
> 
> 2 двоение посмотрим. По умолчанию макс размер 10 мб.

А как расширить до 100мб ? Или это в pro?

**Reply by ttm-alexeypolunin** [reply]
Открыть настройки FPM:

```
nano /etc/php/8.0/fpm/pool.d/totum.conf
```

Изменяем параметр `upload_max_filesize`, сохраняем.

Перезапускаем FPM:

```
service php8.0-fpm restart
```

Открываем настройки nginx:

```
nano /etc/nginx/sites-available/totum.online.conf
```

Изменяем параметр `client_max_body_size` и сохраняем.

Перезапускаем nginx:

```
service nginx restart
```

**Reply by mihail-cornev** [reply]
Подскажите, а почему исчезла возможность увеличиваться изображению, когда увеличиваешь высоту поля?
![Screenshot_14554](https://user-images.githubusercontent.com/113771444/209089243-4fd92d63-f334-46a8-87e1-a6fe0845f08e.jpg)

**Reply by ttm-alexeypolunin** [reply]
По картинке не видно задана ли полю высота в setFloatFormat

Именно полю, а не блоку

**Reply by mihail-cornev** [reply]
> По картинке не видно задана ли полю высота в setFloatFormat
> 
> Именно полю, а не блоку

Подскажите пожалуйста, почему после увеличения высоты поля методом setfloatformat, кликабельность картинки исчезает? Т.е. теперь нельзя нажать на картинку, чтобы посмотреть её в полном размере.
